xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -456,7 +456,7 @@ void CDVDSubtitlesLibass::ApplyStyle(const std::shared_ptr<struct style>& subSty
           ConvColor(subStyle->shadowColor, subStyle->shadowOpacity); // Set the box shadow color
       style->Shadow = (10.00 / 100 * subStyle->shadowSize) * scale; // Set the box shadow size
       // By default a box overlaps the other, then we increase a bit the line spacing
-      lineSpacing = 6.0;
+      lineSpacing = 8.0 * scaleDefault;
```

Comment:
Fine for now but maybe for the future define the magic 8.0 else where. Since this is pretty much an heuristic value maybe allow the user to change it in advanced settings

Reply:
yes we can talk about it (not here)